src/PostmarkTransport.php

```diff
@@ -56,16 +61,29 @@ public function __construct(ClientInterface $client, $key)
         $this->client = $client;
     }
 
+    /**
+     * Get the Postmark API endpoint.
+     *
+     * @param  \Swift_Mime_SimpleMessage  $message
+     * @return string
+     */
+    public function getApiEndpoint(Swift_Mime_SimpleMessage $message)
+    {
+        if ($this->templated($message)) {
+            $this->apiEndpoint.'/withTemplate';
```

Comment:
Should this be `$this->apiEndpoint .= '/withTemplate';`?

Reply:
Nope. Just append it with a `.` 😉

Reply:
Sorry you're right. Had the flue, last week. Think I still need some rest 😉 
Will fix that one.